### PR TITLE
ui: change commentTag style

### LINKED ISSUE #11592: Comment should have multiple tags
chrome_2020-05-24_00-16-01

We should show both [Owner] as well as [Author] for comment tag. Needs refactoring of `commentTag` and related code, currently it supports only one tag for each comment.

https://github.com/go-gitea/gitea/blob/ac3c4677707b7cec3faa42fbf8b5417be193a0ac/routers/repo/issue.go#L653-L668

### PR DESCRIPTION
**Pull request by a1012112796**
* Add comment tag in first comment (issue content)
* Show CommentTagPoster in other way

view:
image

fix #11592

### CODE REVIEW

File: routers/repo/issue.go [hunk 1]
```diff
@@ -909,6 +907,13 @@ func ViewIssue(ctx *context.Context) {
 	// check if dependencies can be created across repositories
 	ctx.Data["AllowCrossRepositoryDependencies"] = setting.Service.AllowCrossRepositoryDependencies
 
+	issue.ShowTag, err = commentTag(repo, issue.Poster, issue)
+	if err != nil {
```
Comment (lafriks) [inline routers/repo/issue.go]:
```suggestion
	if issue.ShowTag, err = commentTag(repo, issue.Poster, issue); err ! = nil {
```

File: models/issue_comment.go [hunk 2]
```diff
@@ -102,7 +102,6 @@ type CommentTag int
 // Enumerate all the comment tag types
 const (
 	CommentTagNone CommentTag = iota
-	CommentTagPoster
```
Comment (lunny) [inline models/issue_comment.go]: No, you cannot delete this line.

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by CirnoT** [comment]
Related https://github.com/go-gitea/gitea/issues/11592

File: models/issue_comment.go [hunk 3]
```diff
@@ -102,6 +102,7 @@ type CommentTag int
 // Enumerate all the comment tag types
 const (
 	CommentTagNone CommentTag = iota
+	CommentTagPoster
```
Comment (lunny) [inline models/issue_comment.go]: I would like put the new enmuerate value to the last.

**Comment by a1012112796** [comment]
ping :)

**Review by lunny — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**